pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/patch/TestParticipantUtil.java

```diff
@@ -0,0 +1,361 @@
+                ResultSet rs = stmt.executeQuery();
+                while (rs.next()) {
+                    OncHistoryDetail oncHistoryDetail = OncHistoryDetail.getOncHistoryDetail(rs);
+                    oncHistoryDetail.setTissues(tissues);
+                    dbVals.resultValue = oncHistoryDetail;
+                }
+            } catch (SQLException e) {
+                throw new RuntimeException(e);
+            }
+            return dbVals;
+        });
+
+        return (OncHistoryDetail) results.resultValue;
+
+    }
+
+    public String createTissueForParticipant(String oncHistoryDetailId) {
+        SimpleResult results = inTransaction((conn) -> {
+            SimpleResult dbVals = new SimpleResult();
+            PreparedStatement stmt = null;
+            try {
+                stmt = conn.prepareStatement(INSERT_TISSUE, Statement.RETURN_GENERATED_KEYS);
+                stmt.setString(1, oncHistoryDetailId);
+                stmt.setLong(2, System.currentTimeMillis());
+                stmt.executeUpdate();
+                ResultSet rs = stmt.getGeneratedKeys();
+                dbVals.resultValue = getPrimaryKey(rs, "tissue_id");
+            } catch (SQLException e) {
+                throw new RuntimeException(e);
+            }
+            return dbVals;
+        });
+
+        return (String) results.resultValue;
+    }
+
+    public String createSmIdForParticipant(String tissueId, String smIdValue) {
+        String smIdUssType = new TissueSMIDDao().getTypeForName("uss");
+        SimpleResult results = inTransaction((conn) -> {
+            SimpleResult dbVals = new SimpleResult();
+            PreparedStatement stmt = null;
+            try {
+                stmt = conn.prepareStatement(INSERT_USS_SM_ID, Statement.RETURN_GENERATED_KEYS);
+                stmt.setString(1, tissueId);
+                stmt.setString(2, smIdValue);
+                stmt.setString(3, smIdUssType);
+                stmt.setLong(4, System.currentTimeMillis());
+                stmt.executeUpdate();
+                ResultSet rs = stmt.getGeneratedKeys();
+                dbVals.resultValue = getPrimaryKey(rs, "sm_id_pk");
+            } catch (SQLException e) {
+                throw new RuntimeException(e);
+            }
+            return dbVals;
+        });
+
+        return (String) results.resultValue;
+    }
+
+    public void deleteSmId(List<String> smIdPKs) {
+        for (String smIdPk : smIdPKs) {
+            try {
+                delete("sm_id", "sm_id_pk", smIdPk);
+            } catch (Exception e) {
+                log.error("unable to delete sm id {}", smIdPk, e);
+            }
+        }
+    }
+
+    public void deleteTissue(List<String> tissueIds) {
+        for (String tissueId : tissueIds) {
+            try {
+                delete("ddp_tissue", "tissue_id", tissueId);
+            } catch (Exception e) {
+                log.error("unable to delete tissue {}", tissueId, e);
```

Comment:
We should probably be throwing exceptions here, rather than silently logging.  Client code will not know that something has gone wrong and they probably should.